src/components/modals/WalletListModal.js

```diff
@@ -292,7 +292,15 @@ class WalletListModalConnected extends React.Component<Props, State> {
   createAndSelectWallet = async ({ currencyCode, walletType }: CreateWalletType) => {
     try {
       const wallet = await this.createWallet(currencyCode, walletType)
-      this.props.bridge.resolve({ walletId: wallet.id, currencyCode: wallet.currencyInfo.currencyCode })
+      const checkWalletsThenSelect = async () => {
+        if (this.props.wallets[wallet.id]) {
+          this.props.bridge.resolve({ walletId: wallet.id, currencyCode: wallet.currencyInfo.currencyCode })
+        } else {
+          await setTimeout(() => {}, 50)
```

Comment:
No need to await. Just set `checkWalletsThenSelect` as the callback of setTimeout.